dashboard/src/components/AppView/ChartInfo.scss

```diff
@@ -2,3 +2,13 @@
   display: block;
   width: 100%;
 }
+
+.ChartInfoUpdate {
+  color: white;
+  width: 100%;
+  margin-bottom: 0.5em;
+  text-align: center;
+  padding: 0.25em 0;
+  font-size: 14px;
+  background: #008145;
```

Comment:
Aren't we using HEx library in this project? I don't remember, honest. You can use the `color` method if it's available.

Reply:
it seems we are not :)